packages/app/src/components/nominationInfo/healthProviderSearch/index.js

```diff
@@ -17,6 +18,7 @@ const SearchHealthProvider = () => {
         return [nomination.providerName, nomination.patientName, nomination.nominationName, nomination.representativeName].some((nom) => nom.includes(searchTerm));
       });
       setSearchHealthcareProvider(filteredNoms);
+      setCount(filteredNoms.length);
```

Comment:
Since this is only used in one place I'm not sure if it makes sense to keep a separate state variable for this. You can reference `SearchHealthcareProvider.length` in the jsx directly.